bucket/charm-gum.json

```diff
@@ -19,13 +21,15 @@
         "architecture": {
             "32bit": {
                 "url": "https://github.com/charmbracelet/gum/releases/download/v$version/gum_$version_Windows_i386.zip",
+                "extract_dir": "gum_0.14.0_Windows_i386",
                 "hash": {
                     "url": "$baseurl/checksums.txt",
                     "regex": "($sha256)\\s+gum_[\\d.]+_Windows_i386\\.zip"
                 }
             },
             "64bit": {
                 "url": "https://github.com/charmbracelet/gum/releases/download/v$version/gum_$version_Windows_x86_64.zip",
+                "extract_dir": "gum_0.14.0_Windows_x86_64",

```

Comment:
And `"extract_dir": "gum_$version_Windows_x86_64"` here.

Reply:
Thanks!